CMSgov/mint-app
# [EASI-1940] - General Characteristics FE

# EASI-1940

## Changes and Description

- Task List section for General Characteristics
- Query for existing models
- Imported MultiSelect component from EASI
- Custom component for AddNote
- Cypress and unit testing

## Notes

There are no validations on the form as none are required as far as I can tell.  This section is completely free form, no dates to validate, etc.  
Ashley's PR on the MultiSelect is still in review.  I wanted to get this PR started, and have implemented the component from EASI with a few adjustments.  I may come back to this pending Ashley's PR doesn't pass review.

## How to test this change

- Create or seed a model plan
- Select `General Characteristics` from the task list
- Try to break it on each page of the form

MultiSelect combobox should be a combination of both current MINT models and existing models imported from external source into db.  Sorted alphabetically.

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
